l10n_it_fatturapa_out/__manifest__.py

```diff
@@ -16,6 +16,7 @@
                'l10n_it_fatturapa_out',
     'license': 'LGPL-3',
     'depends': [
+        'l10n_it_account',
```

Comment:
Eliminerei la dipendenza diretta a `l10n_it_account` in quanto `l10n_it_fatturapa` dipende già da `l10n_it_account`

Reply:
Però esplicito è meglio di implicito. In questo caso usiamo direttamente funzionalità di `l10n_it_account`, quindi penso sia bene esplicitarla.
Anche perchè chi va a leggere il codice potrebbe chiedersi da dove viene `encode_for_export` e se c'è la dipendenza esplicita viene chiarito più facilmente.

Reply:
ok, ma se provi ad aggiornare `l10n_it_account`, quante volte viene aggiornato `l10n_it_account_out` ?

Reply:
`l10n_it_fatturapa_out` viene aggiornato una volta

Reply:
Il comportamento errato di aggiornamento multiplo che, più di una volta hai segnalato tu @vincenzoterzulli, credo sia limitato a versioni vecchie e datate di Odoo.

Per esperienza personale, posso garantire che già dalla versione `10.0` questo problema non sussiste: un modulo viene aggiornato una ed una sola volta, indipendentemente da quante volte venga citato nei `__manifest__` di tutti gli altri moduli.